data/json/snippets/newspapers.json

```diff
@@ -350,6 +362,10 @@
       {
         "id": "weeks_old_news_17",
         "text": "LOCAL EVACUATION ORDERS.  FEMA has announced small-scale evacuations of select centers where rioting is most intense.  \"This is a preventive measure,\" Spokesperson Linda Garrison told our correspondent.  \"We expect to have people home in a few days, but it's best we play it safe until then.\""
+      },
+      {
+        "id": "weeks_old_news_18",
+        "text": "COPS IN CRISIS.  Law enforcement officials report increasing numbers of officers are calling in sick, have resigned, or are just not showing up to duty.  A lack of sufficient officers exacerbates the dangers on duty police face.  In some cases officers are refusing to respond to dispatches in areas with high levels of violence.  A spokesperson for the Worcester County Sheriff's Office confirmed \"It is no longer feasible for us to jeopardize officer lives in some circumstances.  Instead we can continue to rely upon our state of the art AI powered robotic units.\""
```

Comment:
`A lack of sufficient officers` -> `A lack of sufficient personnel`
`on duty police` -> `on-duty police`
`state of the art AI powered robotic units` -> `state-of-the-art AI-powered robotic units`